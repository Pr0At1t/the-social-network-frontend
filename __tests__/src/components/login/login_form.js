import 'react-native';
import React from 'react';
import Login from '../../../../src/components/login/index';

import renderer from 'react-test-renderer';

describe('src/components/login/login_form', function() {
    describe('#render', () => {
        it('renders correctly', () => {
            const tree = renderer.create(<Login />).toJSON();
            expect(tree).toMatchSnapshot();
        });
    });
});
