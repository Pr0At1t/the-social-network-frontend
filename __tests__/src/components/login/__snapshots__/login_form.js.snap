// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/login/login_form #render renders correctly 1`] = `
<RCTScrollView
  automaticallyAdjustContentInsets={false}
  contentContainerStyle={
    Object {
      "backgroundColor": "#3498db",
      "flex": 1,
    }
  }
  contentInset={
    Object {
      "bottom": 0,
    }
  }
  enableAutoAutomaticScroll={true}
  enableResetScrollToCoords={true}
  extraHeight={75}
  extraScrollHeight={0}
  keyboardDismissMode="interactive"
  keyboardOpeningTime={250}
  onScroll={[Function]}
  resetScrollToCoords={
    Object {
      "x": 0,
      "y": 0,
    }
  }
  scrollEnabled={false}
  scrollEventThrottle={0}
  showsVerticalScrollIndicator={true}
  style={
    Object {
      "backgroundColor": "#3498db",
    }
  }
>
  <View>
    <View
      style={
        Object {
          "marginTop": 425,
          "padding": 20,
        }
      }
    >
      <TextInput
        autoCapitalize="none"
        placeholder="username or email"
        placeholderTextColor="rgba(255, 255, 255, 0.7)"
        style={
          Object {
            "backgroundColor": "rgba(255, 255, 255, 0.2)",
            "color": "#FFF",
            "height": 40,
            "marginBottom": 15,
            "paddingHorizontal": 10,
          }
        }
      />
      <TextInput
        placeholder="password"
        placeholderTextColor="rgba(255, 255, 255, 0.7)"
        secureTextEntry={true}
        style={
          Object {
            "backgroundColor": "rgba(255, 255, 255, 0.2)",
            "color": "#FFF",
            "height": 40,
            "marginBottom": 15,
            "paddingHorizontal": 10,
          }
        }
      />
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        collapsable={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        nativeID={undefined}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "backgroundColor": "#2980b9",
            "opacity": 1,
            "paddingVertical": 15,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          disabled={false}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#FFFFFF",
              "fontWeight": "700",
              "textAlign": "center",
            }
          }
        >
          LOGIN
        </Text>
      </View>
    </View>
  </View>
</RCTScrollView>
`;
